INSTALL.rst

```diff
@@ -11,22 +11,49 @@ Prerequisite Software
 ---------------------
 
 HaaS requires a number of packages available through the CentOS RPM
-repositories, as well as the EPEL repository. EPEL can be enabled via:
-
-::
+repositories, as well as the EPEL repository. EPEL can be enabled via::
 
     yum install epel-release
 
-Then, the rest of the packages can be installed via:
+Then, the rest of the packages can be installed via::
 
-::
-
-    yum install libvirt bridge-utils ipmitool telnet sqlite httpd mod_wsgi python-pip qemu-kvm python-virtinst
+    yum install libvirt bridge-utils ipmitool telnet httpd mod_wsgi python-pip qemu-kvm python-virtinst virt-install
 
 In addition, HaaS depends on a number of python libraries. Many of these are
 available as RPMs as well, but we recommend installing them with pip, since
 this will install the versions that HaaS has been tested with.  This is done
-automatically by the instructions below.
+automatically by the instructions below. HaaS supports both SQLite and 
+PostgreSQL. You should install either of the two for HaaS. 
+
+Setting Up HaaS Database
+---------------------
+
+Below are the steps to install PostgreSQL::
+
+    yum -y update
+    yum -y gcc install postgresql postgresql-contrib postgresql-server postgresql-devel postgresql-libs python-psycopg2
+    service postgresql initdb
+    chkconfig postgresql on
+    serivce postgresql start
+
+By default postgresql uses IDENT based authentication. All you have to do is allow username and passowrd based authentication for your network or webserver. Replace 'ident' or 'peer' with 'trust' in the above file (Setup permission to allow postgresql to be accessed by other applicaitons)::
+
+    vi /var/lib/pgsql/data/pg_hba.conf
+
+E.g. "local all all peer" -> "local all all trust";"host all 127.0.0.1/32 ident" -> host all 127.0.0.1/32 trust"; Restart postgresql service once the changes have been made::
+
+   service postgresql restart
+	    
+A default user named 'postgres' is created. Create a database as follows::
+
+    psql -h localhost -U postgres      //Command to enter the psql (postgresql interactive terminal) console
+    create database <db-name>;         //Command to create the database (alway succeeded by a semicolon)
+    \q                                 //Command to quit the database
+
+
+Below are the steps to install and setup SQLite::
+
+    yum -y install sqlite
```

Comment:
sqlite is a dependency of python, so this isn't necessary --- it will have been pulled in automatically.